src/utils/compoundFunctions/index.js

```diff
@@ -0,0 +1,43 @@
+const { flow } = require("lodash/fp");
+const { parseContent, getInnerHTMLWithPiping } = require("../HTMLUtils");
+const convertPipes = require("../convertPipes");
+
+const processPipe = ctx => flow(convertPipes(ctx), getInnerHTMLWithPiping);
+
+const wrapContents = propName => content => {
+  if (!propName || propName === "" || !content) {
+    return;
+  }
+
+  const result = parseContent(content);
+
+  if (!result) {
+    return;
+  }
```

Comment:
Should this throw an error rather than return undefined? Or, handle the error that's thrown from parseContent if you decide to go down that route?